src/core/playback_manager.py

```
@@ -56,6 +56,25 @@ def __init__(
+                            mount_root=mount_root,
+                            media_dir=media_dir,
+                            hint="Run: ./preflight.sh --apply-fixes  (then restart containers)",
+                        )
+                    elif not os.access(media_dir, os.W_OK):
+                        logger.warning(
+                            "Asterisk media directory is not writable; file-based playback may fail",
+                            media_dir=media_dir,
+                            hint="Run: ./preflight.sh --apply-fixes  (then restart containers)",
+                        )
+            except Exception:
+                pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)